docs/pkg-tutorial.md

```diff
@@ -0,0 +1,259 @@
+# Packaging Tutorial
```

Comment:
What's the difference between this file and `pkg-application-tutorial.md`? They seem to be mostly the same, with this file seemingly being a subset of the other?

Reply:
I renamed the file, without removing the orignal copy. I have removed it.